packages/h5p-server/src/types.ts

```diff
@@ -2089,3 +2092,16 @@ export interface ILockProvider {
         options: { timeout: number; maxOccupationTime: number }
     ): Promise<T>;
 }
+
+export interface ITheme {
+    themeUrl: string;
+    fontColor: string;
+    disabledFontColor: string;
+    backgroundColor: string;
+    paperBackgroundColor: string;
```

Comment:
"paper" is a Material UI term; is there a more generic term?

Reply:
secondaryBackground?

Reply:
Ok